camel/logger.py

```diff
@@ -0,0 +1,102 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import logging
+import os
+import sys
+
+# Create a private logger
+_logger = logging.getLogger('camel')
+_logging_disabled = False
+
+
+class PrintLogger(logging.Logger):
+    def _log(self, level, msg, args, **kwargs):
+        if args:
+            msg = msg + ' ' + ' '.join(str(arg) for arg in args)
+        super()._log(level, msg, (), **kwargs)
+
+
+def _configure_library_logging():
+    global _logging_disabled
```

Comment:
maybe it's better not use global, how about encapsulating these in a class?

Reply:
And having a global instance of that class to access, or a singleton? Or should disabling logging not be a global thing? I figured it would be useful to make camel silent in cases where stdout is part of a process

btw I think I've confusingly named this file, it would be better called 'logging.py', which might influence what you think is best.

Reply:
I feel encapsulating the logging configuration in a class and using a singleton pattern could indeed be a good approach. in this way we can maintain the global control but avoid using a global variable directly, WDYT?

Reply:
Makes sense, I'll do it this way. Thanks

Reply:
I've opted to reading and setting an env var instead, since this can allow CAMEL applications to have non-application specific logging disabled from containers. What do you think of this approach?